src/package.json

```diff
@@ -5,7 +5,8 @@
         "build": "vite build",
         "lint": "tsc && prettier --check vite.config.ts resources/js && eslint vite.config.ts resources/js",
         "format": "prettier --write vite.config.ts resources/js",
-        "test": "vitest",
+        "test": "vitest run",
```

Comment:
`vitest run` just runs the tests and exists.

`vitest` runs the tests, watches for changes and then continually re-runs the tests as files change.